network_wrangler/transitnetwork.py

```diff
@@ -1025,22 +1039,17 @@ def _apply_transit_feature_change_routing(
                     sort=False,
                 )
 
-        # Replace self if in_place, else return
-        if in_place:
-            self.feed.shapes = shapes
-            self.feed.stops = stops
-            self.feed.stop_times = stop_times
-        else:
-            updated_network = copy.deepcopy(self)
-            updated_network.feed.shapes = shapes
-            updated_network.feed.stops = stops
-            updated_network.feed.stop_times = stop_times
-            return updated_network
+            net.feed.shapes = shapes
+            net.feed.stops = stops
+            net.feed.stop_times = stop_times
+            return net
```

Comment:
The indentation of this return seems off, should we move it outside of the if?

Reply:
good catch